index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="./style.css">
+    <title>Portfolio Aron</title>
+</head>
+<body>
+    <header class="top-navigation">
+        <a href="#" class="log">Aron Abraham</a>
+        <nav class="item-navigation">
+            <a href="#"><i class="fa fa-bars"></i></a>                    
+        </nav>
+    </header>
+    <section class="container-section">
+        <img src="./Image/Ellipse_18(1).svg" alt="circle" class="image-circle">
+        <div class="container-header">
+            <h1 class="header-text">Aron Abraham</h1>
+            <h2 class="support-text">Software Developer</h2>
+            <p class="text-container">I can help you build a product, feature or website 
+                Look through some of my work and experience! if 
+                you like what you see and have a project you need
+                    coded, don't hestiate to contact me.
+            </p>
+        </div>        
+    </section>
+    <div class="thematic">
```

Comment:
The content rendered starting at this line until the end of the file should be moved inside the `<section>` container.

I would also recommend you don't use the `<main>` tag unless the content you render inside of it actually is the page's main content. Please change all of your `<main>` tags to `<div>` tags to comply with HTML best practices.